konvoy: document details about upgrade with multiple AZs (#4601)
mesosphere/dcos-docs-site: pages/dkp/konvoy/2.1/major-version-upgrade/upgrade/aws/index.md

## Update the cluster worker node pool to Kubernetes version v1.21.6

> 1.  If your cluster is in multipl Availability Zones repeat the following for the number of zones by with values 0,1,2 in `{.items[0].metadata.name}` when exporting `MACHINEDEPLOYMENT_NAME`.

Reviewer: ```suggestion
1.  If your cluster is in multiple Availability Zones, repeat the following for the number of zones, using values 0,1,2 in `{.items[0].metadata.name}` when exporting `MACHINEDEPLOYMENT_NAME`.
```
